docs/tutorials/integrations/kserve-mm/canary.md

```diff
@@ -170,11 +170,11 @@ EOF
 ??? note "About the candidate"
     In this tutorial, the model source (field `storageUri`) for the candidate version is the same as for the primary version of the model. In a real example, this would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
 
-When the candidate version is ready, the Iter8 controller will Iter8 will automatically reconfigure the routing so that inference requests with the header `traffic` set to `true` will be sent to the candidate model. All other requests will be sent to the primary model.
+When the candidate version is ready, the Iter8 controller will Iter8 will automatically reconfigure the routing so that inference requests with the header `traffic` set to `test` will be sent to the candidate model. All other requests will be sent to the primary model.
```

Comment:
Do you explain why it should be set to `test` now?

Reply:
It should have always been `test`. If you look at the example code for sending requests, you will see `test`. See, for example, https://github.com/iter8-tools/docs/blob/main/docs/tutorials/integrations/kserve-mm/canary.md?plain=1#L100